public/js/main-template.js

```diff
@@ -12,93 +12,122 @@ $(document).ready(function () {
+                document.getElementById("pannelone").classList.remove("Active");
+                event.currentTarget.parentElement.classList.remove("RightPanel1Active");
+                document.getElementById("RightPanelone").classList.remove("Active");
+            }
+            event.currentTarget.classList.toggle("Active");
+            event.currentTarget.parentElement.classList.toggle("RightPanel2Active");
+            document.getElementById("installationSteps").classList.toggle("Active");
+
+        }
+        if (event.currentTarget.classList.contains("LeftPannelIcon1")) {
+            event.currentTarget.classList.toggle("Active");
+            event.currentTarget.parentElement.classList.toggle("AsideBarLeftActive");
+        }
     }
-    if (!$(".AsideBarInstallRight").hasClass("RightPanel2Active")) {
-        addClasses([rightMenuBtn], "display-block");
-        removeClasses([rightMenuBtn], "display-none");
+    if (page == "install") {
+        document.getElementById("pannelthree").classList.add("Active");
+        document.getElementById("AsideBarleft").classList.add("AsideBarLeftActive");
+        document.getElementById("panneltwo").classList.add("Active");
+        document.getElementById("AsideBarRight").classList.add("RightPanel2Active");
+        document.getElementById("installationSteps").classList.add("Active");
+        document.getElementById("RightPanelone").classList.remove("Active");
+        document.getElementById("AsideBarRight").classList.remove("RightPanel1Active");
+        document.getElementById("pannelone").classList.remove("Active");
+
     }
-    else {
-        removeClasses([rightMenuBtn], "display-block");
-        addClasses([rightMenuBtn], "display-none");
+    if (page == "field") {
+        document.getElementById("pannelthree").classList.remove("Active");
+        document.getElementById("AsideBarleft").classList.remove("AsideBarLeftActive");
+        document.getElementById("panneltwo").classList.remove("Active");
+        document.getElementById("AsideBarRight").classList.remove("RightPanel2Active");
+        document.getElementById("installationSteps").classList.remove("Active");
+        document.getElementById("RightPanelone").classList.add("Active");
+        document.getElementById("AsideBarRight").classList.add("RightPanel1Active");
+        document.getElementById("pannelone").classList.add("Active");
+
     }
 }
 
-function toggleInstallMenu() {
-    $(".LeftPannelIcon1").toggleClass("Active", { duration: 3000 });
-    removeClasses([leftInstallMenuBtn], "display-none");
-    addClasses([leftInstallMenuBtn], "display-block");
-    removeClasses([rightInstallMenuBtn], "display-none");
-    addClasses([rightInstallMenuBtn], "display-block");
-    $(".AsideBarLeft").toggleClass("AsideBarLeftActive");
 
-    // $(".ContentWrapper").removeClass("ChangeFlex");
+function filterPopup(e) {
+    document.getElementById("FilterModalId").classList.toggle("is-visible") 

```

Comment:
change all the ids to either camelCase or with "-" 
```suggestion
    document.getElementById("filterModalId").classList.toggle("is-visible") 

```